extensions/grpc/codegen/src/main/java/io/quarkus/grpc/deployment/GrpcCodeGen.java

```diff
@@ -146,6 +146,24 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
         return false;
     }
 
+    private static void copySanitizedProtoFile(Path protoPath, Path outProtoPath) throws IOException {
+        try (var reader = Files.newBufferedReader(protoPath);
+                var writer = Files.newBufferedWriter(outProtoPath)) {
+
+            String line = reader.readLine();
+            while (line != null) {
```

Comment:
I believe it would be simpler to use a Stream<String> and just filter out the` java_generic_services` one.

Reply:
If you mean `reader.lines().stream().filter().foreach(line -> writer.write(line))`; Then `IOException `must be catched in the `foreach`. Caller is already catching IOException, I think it's easier to let it bubble up.